### PR TITLE
[hotfix][docs] Fix the dependent version of the hive connector error .

### PR DESCRIPTION
**Pull request by yangjf2019**
## What is the purpose of the change

* Fix the dependent version of the hive connector error.

## Brief change log

*I modified the "docs/dev/table/hive/index.md" and  "docs/dev/table/hive/index.zh.md"  file .*

## Verifying this change

*I have verified the changes by starting the build script in preview mode.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

Hi, @wuchong @TisonKun ,please take a look ,thank you very much !

### CODE REVIEW

File: docs/dev/table/hive/index.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ To integrate with Hive, users need the following dependencies in their project.
 
 <dependency>
   <groupId>org.apache.flink</groupId>
-  <artifactId>flink-shaded-hadoop-2-uber-2.7.5-{{site.version}}</artifactId>
-  <version>{{site.version}}</version>
+  <artifactId>flink-shaded-hadoop-2-uber</artifactId>
+  <version>2.7.5-7.0</version>
```
Comment (wuchong) [inline docs/dev/table/hive/index.md]: Can we auto-generate the shaded version? It's hard to trace the shaded dependency version once the shaded version is bumped.

Comment (yangjf2019) [reply docs/dev/table/hive/index.md]: Thank you, but I don't quite understand what you mean. I want to add it to the "docs/_config.yml" file and use the version number as a variable.

Comment (zentol) [reply docs/dev/table/hive/index.md]: Whoever bumps the flink-shaded version also has to ensure that the documentation is up-to-date.
While it _is_ possibly to extract this property from maven, it doesn't seem worth the trouble. (You'd have to setup something similar to the REST API docs; a generator populates some field in _config.yml, and a test verifies that the value is identical to the current property)

Comment (zentol) [reply docs/dev/table/hive/index.md]: In any case, such a setup would be out-of-scope of this PR.

@yangjf2019 Having a variable in docs/_config.yml for the flink-shaded version sounds like a good idea.

Comment (wuchong) [reply docs/dev/table/hive/index.md]: I think @zentol proposed a good idea. We can create a followup JIRA issue to track this effort. 
And having a flink-shaded version in `docs/_config.yml` as a temporary solution in this PR.

Comment (yangjf2019) [reply docs/dev/table/hive/index.md]: Hi, @wuchong @zentol , Thank you very much ! Someone has committed for this [PR](https://github.com/apache/flink/commit/614e9b73345c3ec09d55f75426968b7329c7953a),Maybe i will closed .

Comment (wuchong) [reply docs/dev/table/hive/index.md]: However, it doesn't fix the shaded version. We still need a pull request to fix it.

cc @bowenli86

Comment (yangjf2019) [reply docs/dev/table/hive/index.md]: Hi, @wuchong @zentol ,I compiled the latest code, however we do not find the maven dependency as follows. 
`<dependency>
  <groupId>org.apache.flink</groupId>
  <artifactId>flink-shaded-hadoop-2-uber</artifactId>
  <version>2.6.5-1.10-SNAPSHOT</version>
  <scope>provided</scope>
</dependency>
`
`<dependency>
  <groupId>org.apache.flink</groupId>
  <artifactId>flink-hadoop-compatibility_1.10-SNAPSHOT</artifactId>
  <version>1.10-SNAPSHOT</version>
  <scope>provided</scope>
</dependency>`
so I think this [PR](https://github.com/apache/flink/commit/614e9b73345c3ec09d55f75426968b7329c7953a) is problematic. Many users will ask why this maven dependency is not found.And I saw that [‘travis-ci’](https://travis-ci.org/apache/flink/builds/578111960?utm_source=github_status&utm_medium=notification) has failed to build. Why can it be merged into master?
I have submitted a new [Pull Request](https://github.com/apache/flink/pull/9591), please take a look, thank you!

**Pull request closed without merging**